common/app/views/fragments/containers/index.scala.html

```diff
@@ -11,7 +11,7 @@
 @* TODO Consolidate this with the master container template to get rid of repetition *@
 
 @defining(page.webTitle) { case (webTitle) =>
-    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty)"
+    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty, adFree = request.headers.keys.exists(_ equalsIgnoreCase "X-Gu-Commercial-Ad-Free"))"
```

Comment:
Actually, at this point, `request` seems to be a plain `play.api.mvc.RequestHeader` so the properties of `RichRequestHeader` are not available. However, it did work for https://github.com/guardian/frontend/pull/17502/files#diff-9e483d0f7f6313cb58e2a1515b4ee26fR194